.github/CODEOWNERS

```diff
@@ -1,7 +1,9 @@
-*       @toposware/protocol
+-       @topos-network/protocol
```

Comment:
What `-` doing in this place ? is it a new filter?

Reply:
I have no idea why the `*` sign was replaced with `-`. Fixed it and removed the unnecessary padding.

Reply:
Prettier is the reason of this change, _pretty_ inconvenient...